<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FightForm</class>
 <widget class="QWidget" name="FightForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QTabWidget" name="tabWidgetFight">
     <property name="styleSheet">
      <string notr="true">QTabWidget::tab-bar {
     alignment: center;
 }</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_15">
      <attribute name="icon">
       <iconset resource="../../../../res/resources.qrc">
        <normaloff>:/icons/magic_wand_2_16px.ico</normaloff>:/icons/magic_wand_2_16px.ico</iconset>
      </attribute>
      <attribute name="title">
       <string>Sorts</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" rowspan="2">
        <layout class="QGridLayout" name="gridLayout_22">
         <item row="0" column="0">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>29</width>
             <height>98</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButtonMoveTopSpell">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../../res/resources.qrc">
             <normaloff>:/icons/arrow_up_16px.ico</normaloff>:/icons/arrow_up_16px.ico</iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButtonMoveDownSpell">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../../res/resources.qrc">
             <normaloff>:/icons/arrow_down_16px.ico</normaloff>:/icons/arrow_down_16px.ico</iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>29</width>
             <height>122</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="QTableWidget" name="tableWidgetSpells">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Sort</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Cible</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Nombre de lancés</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QGridLayout" name="gridLayout_23">
         <item row="0" column="0">
          <spacer name="horizontalSpacer_29">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>48</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButtonAddSpell">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../../res/resources.qrc">
             <normaloff>:/icons/plus_16px.ico</normaloff>:/icons/plus_16px.ico</iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer_30">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>58</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="pushButtonDeleteSpell">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../../res/resources.qrc">
             <normaloff>:/icons/cross_16px.ico</normaloff>:/icons/cross_16px.ico</iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <spacer name="horizontalSpacer_31">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>48</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_16">
      <attribute name="icon">
       <iconset resource="../../../../res/resources.qrc">
        <normaloff>:/icons/cog_16px.ico</normaloff>:/icons/cog_16px.ico</iconset>
      </attribute>
      <attribute name="title">
       <string>Configurations</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_24">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Phase de préparation</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_25">
          <item row="0" column="0">
           <widget class="QLabel" name="label_10">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Placement en début de combat :</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboBoxCloseFight">
            <item>
             <property name="text">
              <string>Aucun</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Bloquer si quelqu'un entre</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Bloquer</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Bloquer l'accès aux spectateurs :</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBoxPositionFight">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <item>
             <property name="text">
              <string>Aucun</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Proche des ennemies</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Loin des ennemies</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Bloquer l'accès au combat :</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="comboBoxSpectator">
            <item>
             <property name="text">
              <string>Aucun</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Bloquer si quelqu'un entre</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Bloquer</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Combat</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Rapidité en combat :</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="comboBoxSpeedFight">
              <property name="currentIndex">
               <number>1</number>
              </property>
              <item>
               <property name="text">
                <string>Lente</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Normale</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Rapide</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Ultra rapide</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Suicidaire</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Comportement :</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="comboBoxBehavior">
              <property name="currentIndex">
               <number>2</number>
              </property>
              <item>
               <property name="text">
                <string>Fuillard</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Aggressif</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Passif</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Régénération de vie</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_28">
          <item row="0" column="2">
           <widget class="QLabel" name="label_23">
            <property name="text">
             <string>se régénérer jusqu'à</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Si les points de vie sont inférieurs à</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QSpinBox" name="spinBoxRegenerationMax">
            <property name="suffix">
             <string>%</string>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spinBoxRegenerationMin">
            <property name="suffix">
             <string>%</string>
            </property>
            <property name="value">
             <number>36</number>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer_11">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidgetFight</tabstop>
  <tabstop>tableWidgetSpells</tabstop>
  <tabstop>pushButtonMoveTopSpell</tabstop>
  <tabstop>pushButtonMoveDownSpell</tabstop>
  <tabstop>pushButtonAddSpell</tabstop>
  <tabstop>pushButtonDeleteSpell</tabstop>
  <tabstop>comboBoxPositionFight</tabstop>
  <tabstop>comboBoxCloseFight</tabstop>
  <tabstop>comboBoxSpectator</tabstop>
  <tabstop>comboBoxSpeedFight</tabstop>
  <tabstop>comboBoxBehavior</tabstop>
  <tabstop>spinBoxRegenerationMin</tabstop>
  <tabstop>spinBoxRegenerationMax</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
